src/MongoDB.Driver.Legacy/Builders/IndexKeysBuilder.cs

```diff
@@ -261,6 +271,30 @@ public override BsonDocument ToBsonDocument()
             return _document;
         }
 
+        /// <summary>
+        /// Sets a wildcard key to the index. If the wildcard name is empty, the generated key will be for the whole collection.
+        /// </summary>
+        /// <param name="name">The wildcard key.</param>
+        /// <returns>The builder (so method calls can be chained).</returns>
+        public IndexKeysBuilder Wildcard(string name = null)
+        {
+            string wildcard = name;
```

Comment:
I guess a new variable is OK. But now we have to be careful never to use the name variable again in the rest of the method.

Can you use var?

Reply:
Done